src/app/containers/Timestamp/relativeTimestamp.test.js

```diff
@@ -0,0 +1,61 @@
+import relativeTimestamp from './relativeTimestamp';
+
+const timeStampGenerator = (timeDifference, magnitude) =>
+  Date.now() - timeDifference * magnitude;
+
+const timestampWithDiffInHours = hours =>
+  timeStampGenerator(hours, 60 * 60 * 1000);
+
+const timestampWithDiffInMins = mins => timeStampGenerator(mins, 60 * 1000);
+
+const relativeBehaviour = (description, input, expectedOutput) => {
+  it(description, () => {
+    const result = relativeTimestamp(input);
+    expect(result).toEqual(expectedOutput);
+  });
+};
+
+describe('relativeTimestamp', () => {
+  it('returns a string which ends in ago', () => {
+    const timestamp = timestampWithDiffInMins(1);
+    const result = relativeTimestamp(timestamp);
+    expect(typeof result).toEqual('string');
+    expect(result.split(' ').pop()).toEqual('ago');
+  });
+
+  relativeBehaviour(
```

Comment:
Probably not needed, but can we have a test for like 30 seconds ago? Also one for in the future to ensure we catch the edge case